src/styles/index.scss

```diff
@@ -1,3 +1,3 @@
-body {
+.body {
```

Comment:
The `.body` class seems unnecessary as the `body` element is already styled in `main.scss`. Consider removing this redundant class.